ClientCredentialsKeypairs/AuthHeaderHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Fhi.ClientCredentialsKeypairs;
+
+/// <summary>
+/// (legacy?) Auth handler in use by 
+/// Fhi.Koronasertifikat.QrPdfApi.Api.IntegrationTests
+/// Fhi.Grunndata.PersonoppslagApi.IntegrationTests 
+/// Fhi.Grunndata.OppslagAdmin
+/// some people use this one when creating new HttpClient manually..
+/// </summary>
+[Obsolete("Use HttpAuthHandler and assign InnerHandler = new HttpClientHandler() manually.")]
```

Comment:
Do you know why we have this AuthHeaderHandler @OsirisTerje ? I have made some assumptions here, but I don't think we should duplicate our code for this case.